CRM/Event/BAO/Participant.php

```diff
@@ -1845,6 +1845,8 @@ public static function getAdditionalParticipantUrl($participantIds) {
    *   ID of the civicrm_price_field_value field for the discount id.
    */
   public static function createDiscountTrxn($eventID, $contributionParams, $feeLevel, $discountedPriceFieldOptionID = NULL) {
+    // @fixme: Not sure that transaction is required here - but it was previously wrapping this function so moved here
+    $transaction = new CRM_Core_Transaction();
```

Comment:
@mattwire  I think we should remove this altogether - see https://github.com/civicrm/civicrm-core/pull/15460